src/main/java/com/faforever/server/entity/Login.java

```diff
@@ -48,4 +49,8 @@
 
   @Transient
   private String country;
+
+  @Transient
+  @Nullable
+  private String timezone;
```

Comment:
Because it's "time zone" and not "timezone" it should be `timeZone` :)
And Since there's a class `TimeZone` I'd prefer to use it here.